lib/util.js

```diff
@@ -42,10 +42,16 @@ const {
 const {
   deprecate,
   getSystemErrorName: internalErrorName,
-  isError,
   promisify,
 } = require('internal/util');
 
+const ReflectApply = Reflect.apply;
```

Comment:
Why the alias of one name to a nearly identical name, is it because `=> Reflect.apply(` would be a syntax error? I just tried in the console, and it seemed to work OK. Or is it an optimization to avoid a lookup in the Reflect object?

Reply:
it's to keep the value in case userland changes it